### PR TITLE
feat: added transcript in video table [7891]

### LINKED ISSUE #7891: [YouTube] Extract the transcript of a video
MindsDB integrates with YouTube, so you can access YouTube data and make relevant predictions with MindsDB.

## Resources

Familiarize yourself with the existing YouTube integration: https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/youtube_handler

## Steps to Follow

The task is to extend [this implementation](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/youtube_handler) to include the transcript of a video.

Currently, the [YoutubeVideoTable](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/youtube_handler/youtube_tables.py#L206) includes columns such as title, description, etc. The task is to add a transcript column that contains the transcript of the video.

## :loudspeaker: Additional rewards :trophy:

Each integration extension issue brings 15+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left:

### PR DESCRIPTION
**Pull request by rawatsushil**
## Description

This enable youtube handler to get transcript of a video using transcript api

Fixes #7891 

## Type of change

(Please delete options that are not relevant)

- [x] ⚡ New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.

### CODE REVIEW

**Comment by rawatsushil** [comment]
@martyna-mindsdb Can you guide me here, I think the test failures are not because of my changes

File: mindsdb/integrations/handlers/youtube_handler/requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 google-api-python-client
+youtube_transcript_api
```
Comment (martyna-mindsdb) [inline mindsdb/integrations/handlers/youtube_handler/requirements.txt:2]: Shouldn't it be with `-` instead of `_`? Like this, `youtube-transcript-api`.

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   **sushil rawat** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@rawatsushil 
Please complete the following to have this PR merged:

1. As it is your first PR in MindsDB repo, please sign our CLA by leaving a comment here that contains this line: I have read the CLA Document and I hereby sign the CLA

2. See the review comment, implement the correction, and push it here.

3. Provide a screenshot or a short video that presents querying the video table and getting the transcript as one of the output columns.

**Comment by rawatsushil** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by rawatsushil** [comment]
recheck

**Comment by rawatsushil** [comment]
@martyna-mindsdb Hey, can you help me with the steps to be done to setup database connection post which i can query the database .
Readme.md in this youtube handler is not self sufficient for first timers i think.

**Comment by martyna-mindsdb** [comment]
@rawatsushil 
Have installed MindsDB locally? [instruction](https://docs.mindsdb.com/contribute/install)

Then you should install dependencies running `pip install mindsdb[youtube]`.

Now start MindsDB and connect to yt using CREATE DATABASE.

**Comment by rawatsushil** [comment]
Tried first step via pypi and via docker
1) via pypi
Screenshot 2023-10-20 at 9 52 17 PM
But post that unable to connect my sqlworkbench to mindsdb on localhost

2) via docker
Screenshot 2023-10-20 at 10 31 28 PM
same thing happened here as well. Not able to connect my sql workbench to mindsdb on localhost.

i am using "password": "",   "port": "47335",   "user": "mindsdb",

**Comment by martyna-mindsdb** [comment]
@rawatsushil 
It looks like you installed MindsDB via pip - in the screenshot it says `GUI available at ...` so you can access it there.

Follow the remaining steps (like installing dependencies) to complete testing.

**Comment by ZoranPandovski** [comment]
@rawatsushil Do you have an update for this? Please make sure you share the screenshots or video of the working implementation

**Comment by rawatsushil** [comment]
@ZoranPandovski  Sorry forgot to update here, I was facing problem testing it. Though i did the changes but it is not tested yet. Give me a days time. Let me see if i can get it done if i try again

**Comment by rawatsushil** [comment]
I am unable to continue on this, sorry. It could take me some time. I don't want to hold this. 
You can please assign it to somebody else
@ZoranPandovski

**Comment by adeyinkaezra123** [comment]
@rawatsushil @ZoranPandovski, I'd like to work on this as it is currently a blocker for a mindsdb app I'm building

**Comment by adeyinkaezra123** [comment]
@rawatsushil @ZoranPandovski, I'd like to work on this as it is currently a blocker for a mindsdb app I'm building

**Comment by martyna-mindsdb** [comment]
@adeyinkaezra123 
You can complete this PR or create a new one.

**Pull request closed without merging**